WebApplication/Services/LoggedInUserBucketKeyProvider.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Threading.Tasks;
+using Autodesk.Forge.Core;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using WebApplication.Utilities;
+
+namespace WebApplication.Services
+{
+    public class LoggedInUserBucketKeyProvider : IBucketKeyProvider
```

Comment:
What is the point of having LoggedInUserBucketKeyProvider as a base class for MigrationBucketKeyProvider?

Reply:
Shared method GetBucketPrefix. Moved to the separate provider.